.eslintrc.json

```diff
@@ -74,32 +67,48 @@
         "id-blacklist": "off",
         "id-match": "off",
         "max-len": "off",
-        "linebreak-style": [
+        "linebreak-style": ["error", "unix"],
+        "no-underscore-dangle": "off",
+        "@nrwl/nx/enforce-module-boundaries": [
           "error",
-          "unix"
-        ],
-        "no-underscore-dangle": "off"
+          {
+            "allow": [],
+            "depConstraints": [
+              {
+                "sourceTag": "*",
+                "onlyDependOnLibsWithTags": ["*"],
+                "notDependOnLibsWithTags": ["type:app"]
+              },
+              {
+                "allSourceTags": ["type:ui", "scope:core"],
+                "notDependOnLibsWithTags": ["type:feature"]
```

Comment:
That might be subjective. but for readability, I'd have a single config for core below and specify the list of libs it can't depend on ( which is all of them in the case of core. )

Reply:
that's fine with me! fixing!
I just wanted to have an all tags as an example

we can always have all and a specific one which will join them in the end